nbextensions/editorCell/main.js

```diff
@@ -370,12 +358,12 @@ define(
 
             // Set the notebook environment.
             // For instance, we don't want to override the toolbar in the Narrative, but we need to supply our on in a plain notebook.
-            env = 'narrative';
+            // env = 'narrative';
 
             // And make a toolbar preset composed of the extensions, and activate it for this notebook.
-            if (env !== 'narrative') {
-                activateToolbar();
-            }
+            // if (env !== 'narrative') {
+            //     activateToolbar();
+            // }
```

Comment:
Even though these lines do nothing, they looked important due to the comments above them. I left them in but commented them out. Luckily, the comprehensive tests for this module will catch any changes in behaviour.

Reply:
😅